documentation/docs/03-loading.md

```diff
@@ -29,6 +27,8 @@ type LoadOutput = {
 };
 ```
 
+Our example blog page might contain a `load` function like the following. Note the `context="module"` — this is necessary because `load` runs before the component is rendered:
```

Comment:
I agree with moving this. I actually made this change in a few places. If you rebase against `master` it was updated a couple weeks ago in https://github.com/sveltejs/kit/commit/9701eb21b7698226ad287c7ce0f1893ffa6fb972#diff-820e98511cb87c8891f398a00ff2f70793de22e53396d0a3ac6bc2cd6cc3e56e